### PR TITLE
Adds optional language menu, fixes locale settings in demo

### PR DESCRIPTION
**Pull request by farridav**
- Adds an optional language chooser dropdown
- fixes new chinese locales

### CODE REVIEW

File: docs/configuration.md [hunk 1]
```diff
@@ -226,6 +228,24 @@ Puts fieldsets and inlines into a bootstrap carousel, and allows paginaton with
...
+## Language Chooser
+You can enable a language chooser dropdown using `"language_chooser": True` in your `JAZZMIN_SETTINGS`, we mainly use this for 
+assisting with translations, but it could be of use to some people in their admin site.
+
+To make proper use of this, please ensure you have internationalisation setup properly, See https://docs.djangoproject.com/en/3.1/topics/i18n/translation/
+
+Namely:
+
+- i18n urls for your admin
+- `LocaleMiddlware` used
```
Comment (TysonRV) [inline docs/configuration.md:240]: Typo here!

File: jazzmin/templates/admin/base.html [hunk 2]
```diff
@@ -99,6 +99,24 @@
                 </li>
                 {% endif %}
 
+                {% if jazzmin_settings.language_chooser %}
+                    {% get_available_languages as LANGUAGES %}
+                    {% get_language_info_list for LANGUAGES as languages %}
+
+                    <li class="nav-item dropdown">
+                        <a class="nav-link btn" data-toggle="dropdown" href="#" title="Choose language">
+                            <i class="fas fa-globe-americas pr-2" aria-hidden="true"></i>
```
Comment (TysonRV) [inline jazzmin/templates/admin/base.html:108]: I would rather use `<i class="fas fa-flag"></i>` here

**Review by TysonRV — COMMENTED** [review]
This looks really awesome and snappy as hell! Good to go, just a couple of comments.

Great job 🎏

File: jazzmin/templates/admin/base.html (hunk 2 above)
Comment (farridav) [reply jazzmin/templates/admin/base.html]: Unfortunately, the flag does not look like a language icon, more of a racecar icon, ive changed it to `fas fa-globe` so its not biased, though that icon did look most recognisable as a language setting icon

File: docs/configuration.md (hunk 1 above)
Comment (farridav) [reply docs/configuration.md]: done

**Pull request merged**